modules/storages/lib/api/v3/file_links/file_link_relation_representer.rb

```diff
@@ -0,0 +1,77 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2022 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+module API
+  module V3
+    module FileLinks
+      module FileLinkRelationRepresenter
```

Comment:
If one can think of a better name here, please suggest. I also had s.th. like `FileLinkRelationableRepresenter` in mind, to emphasize, that it is not a entity by its own.

Reply:
Not sure, but what about `FileLinkRepresenter`. It would name what it does. If it's included in another representer or not should not make a difference. Or is it always a relation and therefore needs to be named like one?

Reply:
The `FileLinkRepresenter` represents a `FileLink` resource, and would not enrich another resource with links to file links. That's, what I wanted to put into that name. We already have a `FileLinkRepresenter` of course, as there are endpoints delivering file link resources in their responses.